docs/installation.rst

```diff
@@ -133,7 +133,7 @@ Casperjs installation additions
 
 .. versionadded:: 1.1-beta3
 
-- Append ``";C:\casperjs\bin"`` to your ``PATH`` environment variable.
+- Append ``";C:\casperjs\batchbin"`` (versions < 1.1beta3) or ``;C:\casperjs"`` (versions >=1.1 beta3) to your ``PATH`` environment variable.
```

Comment:
It would make it a bit clearer if the wording would be `versions before` and `versions since`